requirements.txt

```diff
@@ -1,11 +1,12 @@
 git+https://github.com/simphony/osp-core.git#egg=osp-core
-sphinx
-recommonmark
-sphinx_rtd_theme
-sphinxcontrib-plantuml
-nbsphinx
-sphinx-copybutton
-ipython
-jupyter
-sphinx-jsonschema
-sphinx-autobuild
+sphinx==3.5.3
+recommonmark==0.7.1
+sphinx_rtd_theme==0.5.2
+sphinxcontrib-plantuml==0.20.1
+nbsphinx==0.8.2
+sphinx-copybutton==0.3.1
+ipython==7.22.0
+jupyter==1.0.0
+sphinx-jsonschema==1.16.8
+sphinx-autobuild==2021.3.14
+git+https://github.com/readthedocs/sphinx-hoverxref@master
```

Comment:
I think the plugin should be [available on PiPy](https://pypi.org/project/sphinx-hoverxref/)

Reply:
Yes, that is true. But the extension is still under active development, and the developer, therefore, advise to use the latest version, which includes additional performance enhancements and bug fixes (see [here](https://github.com/readthedocs/sphinx-hoverxref)):
> This extension is currently in Beta state.